### DISCUSSION TITLE
Add Watopia's Southern Coast Road

### DISCUSSION THREAD

**Discussion by fatphil2001** [sandermvanvliet/RoadCaptain#136] [Ideas]
I was about to build a combination of Watopia's Waistband and Sugar Cookie, but it's not yet possible.  Any thoughts about adding this to the Road Captain route builder? https://zwiftinsider.com/southern-coast-expansion/

**Comment by sandermvanvliet** [comment]
It’s already in the latest pre-release that you can find [here](https://github.com/sandermvanvliet/RoadCaptain/releases)
